### PR TITLE
Release v0.7

### PR DESCRIPTION
**Pull request by jm-clius**
This PR relates to #808

It adds the release notes for nim-waku release v0.7. Please go over the list and LMK if there's anything that I've missed. As with the previous release, I've added a short description of the main benefits users can expect from the release.

### CODE REVIEW

**Review by staheri14 — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,23 +1,45 @@
-##  Next version
+##  2021-01-19 v0.7
 
-This release contains the following:
+Some useful features and fixes in this release, include:
```
Comment (oskarth) [inline CHANGELOG.md:3]:
```suggestion
Release highlights:

```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,23 +1,45 @@
...
+- support for secure websockets
```
Comment (oskarth) [inline CHANGELOG.md:4]:
```suggestion
- Support for secure websockets
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,23 +1,45 @@
...
+- the ability to remove unreachable clients in a `filter` node
```
Comment (oskarth) [inline CHANGELOG.md:5]:
```suggestion
- Ability to remove unreachable clients in a `filter` node
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,23 +1,45 @@
...
+- several fixes to improve `store` performance and decrease query times
```
Comment (oskarth) [inline CHANGELOG.md:6]:
```suggestion
- Several fixes to improve `store` performance and decrease query times
```

Maybe mention specific numbers?

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,23 +1,45 @@
...
+- a fix for a long-standing bug that prevented proper database migration in some deployed Docker containers
```
Comment (oskarth) [inline CHANGELOG.md:7]:
```suggestion
-Fix for a long-standing bug that prevented proper database migration in some deployed Docker containers
```

File: CHANGELOG.md (hunk 1 above)
Comment (oskarth) [inline CHANGELOG.md:3]: Can also consider adding (issue) or (pr) after for people who want more details. Or not, just an idea

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,23 +1,45 @@
...
-- Store: timestamps of message reception, used for indexing, now have consistent millisecond resolution.
+- Increased maximum length for reading from a libp2p input stream to allow largest possible protocol messages, including `HistoryResponse` messages at max size
+- Significantly improved `store` node query performance
+- Implemented a GossipSub `MessageIdProvider` for `11/WAKU2-RELAY` messages instead of relying on the unstable default
+- Receiver timestamps for message indexing in the `store` now have consistent millisecond resolution
+
+This release supports the following [libp2p protocols](https://docs.libp2p.io/concepts/protocols/):
+| Protocol | Spec status | Protocol id |
+| ---: | :---: | :--- |
+| [`17/WAKU-RLN`](https://rfc.vac.dev/spec/17/) | `raw` | `/vac/waku/waku-rln-relay/2.0.0-alpha1` |
```
Comment (oskarth) [inline CHANGELOG.md:35]: 17/WAKU-RLN-RELAY?

**Review by oskarth — APPROVED** [review]
LGTM!

File: CHANGELOG.md (hunk 1 above)
Comment (jm-clius) [reply CHANGELOG.md]: Done in release notes.

**Pull request merged**